Student_BCOs/Christian_ZevallosDelgado_BCO_000515.json

```diff
@@ -1,27 +1,68 @@
 {
-    "bco_id": "http://biocomputeobject.org/BCO_000515",
-    "checksum": "600ab8d367cb739fe6053c903f356737e9bace86131bedf6fe3702214f7dc1a6",
+    "bco_id": "http://biocomputeobject.org/BCO_000547",
+    "checksum": "ddd5b472b12782ba3d74464c8b0c92ea76e76012443409591f043c9e05f97e2b",
     "bco_spec_version": "1.3.0",
     "provenance_domain": {
         "name": "RNA-seq data of invasive ductal carcinoma and adjacent normal tissues from a Korean patient with breast cancer",
-        "version": "1.0.0",
+        "version": "1.1",
         "license": "https://creativecommons.org/licenses/by/4.0/legalcode",
-        "created": "2019-10-06T03:39:59.203155",
-        "modified": "2019-10-08T15:33:06.958657",
+        "created": "2019-10-16T03:53:26.964099",
+        "modified": "2019-10-16T13:33:14.244924",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "2019-10-15"
+            }
+        ],
         "contributors": [
             {
                 "name": "Christian Zevallos Delgado",
                 "orcid": "https://orcid.org/0000-0001-7078-3782",
-                "affiliation": "The George Washington University",
+                "affiliation": "George Washington University",
                 "contribution": [
                     "createdBy"
                 ],
                 "email": "czevallosd@gwu.edu"
+            },
+            {
+                "name": "Jihyoung Hyung Hong",
+                "orcid": "https://orcid.org/0000-0002-0492-3959",
+                "affiliation": "Division of Oncology, Department of Internal Medicine, College of Medicine, The Catholic University of Korea, Seoul, Republic of Korea",
+                "contribution": [
+                    "authoredBy"
+                ]
+            },
+            {
+                "name": "Yoon Ho Ko",
+                "orcid": "https://orcid.org/0000-0002-2506-3740",
+                "affiliation": "Division of Oncology, Department of Internal Medicine, College of Medicine, The Catholic University of Korea, Seoul, Republic of Korea",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112